packages/portal/src/components/stories/StoriesTagsDropdown.vue

```diff
@@ -30,7 +33,7 @@
           :aria-controls="showDropdown ? 'tags-options' : null"
           :aria-expanded="showDropdown"
           :aria-label="$t('categories.label')"
-          @focus="showDropdown = true"
+          @keydown.down="handleInputDown"
```

Comment:
This only works for moving from the input to the first tag, and sometimes scrolls a bit (as that is the native behaviour for the up/down arrows in this case). I'm doubting if that could be confusing. In the main search form the options are ordered top to bottom and don't have a scrollbar. In the search facet dropdowns the up and down arrows navigate through the options instead of control the scroll bar. In the stories tag dropdown I don't think we want either as there are many options to key through before scrolling would happen. So I'd say remove this event listener.

Reply:
yes, agreed, it's not the best. Will remove.

Reply:
removed.